src/main/kotlin/racing/Racing.kt

```diff
@@ -0,0 +1,22 @@
+package racing
+
+import racing.ui.ResultView
+
+object Racing {
+    fun prepareRacing(carsName: List<String>): List<Car> {
+        return carsName.map { Car(it, 0) }
+    }
+
+    fun startRacing(cars: List<Car>, movable: () -> Boolean) {
+        for (car in cars) {
+            move(movable, car)
+        }
```

Comment:
아마 move() 메서드로 분리한 이유는 미션 depth 조건 때문이실까요? 🤔
아래와 같은 방식으로 변경하게되면 move 함수로 굳이 쪼갤 필요가 없을 것 같네요. 😉

```suggestion
        cars.filter { movable() }
            .forEach { it.move() }
```